docs/release-process.md

```diff
@@ -0,0 +1,184 @@
+# How we release c2pa-rs and related crates
+
+The build process for the crates in this repo is automated using [release-plz](https://release-plz.dev), a Rust-specific CI tool.
+
+Release-plz leverages information contains in the various `Cargo.toml` files and its own [`release-plz.toml`](../release-plz.toml) configuration file. 
+
+There are many options available for [configuring release-plz](https://release-plz.dev/docs/config). In this document, we will talk only about how it is configured for this project. For more detail about how release-plz works, we recommend reading the [release-plz documentation site](https://release-plz.dev/docs), which is quite extensive.
+
+## How it works
+
+We use the pre-packaged [GitHub Actions wrapper](https://github.com/release-plz/action) to trigger builds. This is configured in our [`release.yml`](../.github/workflows/release.yml) task.
+
+For additional information, please read the [release-plz documentation for GitHub Action](https://release-plz.dev/docs/github/quickstart).
+
+In this repo, the build automation is triggered for each commit to the `main` branch.
+
+This task runs using a GitHub access token tied to Eric's `@scouten-adobe` account, which is why all PRs and releases are under Eric's name. (See ticket CAI-7621 for moving this to a team-owned bot account.)
+
+### Create a release pull request
+
+This runs [`release-plz release-pr`](https://release-plz.dev/docs/usage/release-pr). For each published crate in the repo, it looks for commits since the last tagged release (ignoring commits with the `chore:` prefix). If it finds any, it will open a new release pull request or update the existing release. (NOTE: When Colin's [PR #2196 to release-plz](https://github.com/release-plz/release-plz/pull/2196) lands there will be a new trigger here which will reference intra-project dependency updates even if the downstream project has no commits of its own.)
```

Comment:
Does it ignore `docs` commits too?

Reply:
No, `docs` commits are included. I've rephrased this a bit, but I want to avoid duplicating the entire list here.